imap_processing/tests/ultra/unit/test_ultra_l1b_extended.py

```diff
@@ -734,12 +734,19 @@ def test_is_back_tof_valid(test_fixture, ancillary_files):
     df_filt, _, _, de_dataset = test_fixture
     df_ph = df_filt[np.isin(df_filt["StopType"], [StopType.PH.value])]
 
+    # Get the ph inds
+    ph_indices = np.nonzero(
+        np.isin(de_dataset["stop_type"], [StopType.Top.value, StopType.Bottom.value])
+    )[0]
```

Comment:
Could this be written as the following using concatenation instead of several numpy calls? I'm not sure it is more efficient, but reads a bit clearer to me than the nonzero version.
```suggestion
    ph_indices = (de_dataset["stop_type"] == StopType.Top.value) | (de_dataset["stop_type"] == StopType.Bottom.value])
```